src/social-network-provider/facebook.com/UI/injectPostBox.tsx

```diff
@@ -13,6 +13,7 @@ if (isMobileFacebook) {
 } else {
     composeBox = new LiveSelector()
         .querySelectorAll('form [role="button"]')
+        .filter((x) => x.textContent === 'Post')
```

Comment:
Hmm, I'm sorry but that doesn't work. Facebook has multiple language support, if you only recognize "Post", you will miss support for other languages.

Reply:
Update, `querySelectorAll('form [role="button"][tabindex="-1"]')` seems won't break other place and solve this.